providers/tentacle.rb

```diff
@@ -168,7 +169,11 @@
     notifies :restart, "windows_service[#{service_name}]", :delayed
   end
 
-  new_resource.updated_by_last_action(actions_updated?([register_instance]))
+  service = windows_service service_name do
```

Comment:
I believe we can ignore this part because if the action above happens then the restart will happen and therefore the resource will be updated.